### PR TITLE
Add support for aggregated grouped_lights commands

### PR DESCRIPTION
**Pull request by marcelveldt**
No more need to send light commands to each individual light of a group as Signify added support to send aggregated commands on the API.

### CODE REVIEW

File: aiohue/v2/controllers/groups.py [hunk 1]
```diff
@@ -86,12 +86,9 @@ def get_lights(self, id: str) -> List[Light]:
...
     async def set_state(self, id: str, on: bool = True) -> None:
-        """
-        Set supported feature(s) to grouped_light resource.
-
-        NOTE: a grouped_light can only handle OnFeature
-        To send other features, you'll have to control the underlying lights
-        """
+        """Set supported feature(s) to grouped_light resource."""
+        # Sending (color) commands to grouped_light was added in Bridge version 1.50.1950111030
+        self._bridge.config.require_version("1.50.1950111030")
```
Comment (balloob) [inline aiohue/v2/controllers/groups.py:91]: Should we make this a decorator ?

**Review by balloob — APPROVED** [review]

File: aiohue/v2/controllers/groups.py (hunk 1 above)
Comment (marcelveldt) [reply aiohue/v2/controllers/groups.py:91]: Yeah I was thinking in that direction too but we require the config object so we'll have to fuss with a lookup from the class instance... Maybe for later ?

**Comment by jdiegmueller** [comment]
> No more need to send light commands to each individual light of a group as Signify added support to send aggregated commands on the API.

I am still seeing the one-at-a-time behavior with HomeAssistant 2022.3.7/aiohue 4.4.1.  Is this expected?

**Comment by marcelveldt** [comment]
> > No more need to send light commands to each individual light of a group as Signify added support to send aggregated commands on the API.
> 
> I am still seeing the one-at-a-time behavior with HomeAssistant 2022.3.7/aiohue 4.4.1. Is this expected?

Yes, you'll have to wait for a new version (2022.4) of HA as this is a new feature.

**Pull request merged**